.gitignore

```diff
@@ -91,3 +91,7 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+# IDE files
+.vscode/
+.idea/
```

Comment:
Just realized this line was repeated, it's already in line 26

Reply:
Nice observation Brian-Cee. On it